drivers/hmis_data_quality_tool/app/models/hmis_data_quality_tool/report.rb

```diff
@@ -380,6 +380,154 @@ def average_days_before_entry
       sum / count
     end
 
+    def average_days_to_enter_entry_date
+      count = enrollments.count
+      sum = enrollments.sum(:days_to_enter_entry_date)
+
+      return 0 if count.zero?
+
+      sum / count
```

Comment:
these are integers. That means the value will be truncated (not rounded). is that expected? (same on line 380, 400, and 480)
```ruby
> (3/2)
=> 1
(3.0/2).round
=> 2
```

I guess I would expect a float here
```suggestion
      sum.to_f / count
```

Reply:
Good catch, thank you!